docs/docs/rest-api/mesosphere/marathon/api/v2/AppsResource_show.md

```diff
@@ -181,6 +181,17 @@ Transfer-Encoding: chunked
                 "portIndex": 0,
                 "protocol": "HTTP",
                 "timeoutSeconds": 10
+            },
+            {
+                "command": null,
+                "gracePeriodSeconds": 5,
+                "intervalSeconds": 10,
+                "maxConsecutiveFailures": 6,
+                "path": "/machinehealth",
+                "portIndex": 0,
```

Comment:
This actually does show, since portIndex is not nullable.

This also shows on "protocol": "COMMAND": https://gist.github.com/EvanKrall/026de225cabe1843762e